extensions/go-links/CHANGELOG.md

```diff
@@ -0,0 +1,37 @@
+# Changelog
```

Comment:
Changelog should look something [like this](https://raw.githubusercontent.com/raycast/extensions/9cf6a66f16f53e069a5032193b9b930bf52f1237/extensions/tableplus/CHANGELOG.md)

Reply:
The [Version History](https://developers.raycast.com/basics/prepare-an-extension-for-store#version-history) section of your documentation references [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), which is the format that I followed here. Is that not accurate anymore?

<img width="782" alt="Screenshot 2024-10-01 at 8 38 44 AM" src="https://github.com/user-attachments/assets/dc9c1ecd-fbc9-4924-8c34-96ffd340c2f5">